pages/en/anime/watch/[...info].js

```diff
@@ -474,6 +485,23 @@ export default function Watch({
           content={episodeNavigation?.playing?.description || info?.description}
         />
       </Head>
+      <RateModal
+        onSubmit={async (e) => {
+          e.preventDefault();
+          const rating = e.target.rating.value;
+          const review = e.target.review.value;
+          await markProgress({
+            mediaId: dataMedia.id,
+            progress: episodeNavigation.playing.number,
+            notes: review,
+            // scoreRaw is 1-100 while our input is 1-10
+            rating: 10 * rating,
+          });
+          toast.success(`Successfully submitted your review and rating.`);
```

Comment:
this toast should be inside the function because this will always running even if something happen when mutating the data

Reply:
I'm assuming you mean that you want there to be an error toast incase it fails, just added a catch for that